packages/backend/src/services/GithubAppService/GithubAppService.ts

```diff
@@ -91,6 +91,60 @@ export class GithubAppService extends BaseService {
                 throw new ParameterError('User uuid not provided');
             }
 
+            const redirectUrl = new URL(
+                oauth?.returnTo || '/generalSettings/integrations',
+                this.lightdashConfig.siteUrl,
+            );
```

Comment:
Does this change the previous behavior? If no `returnTo` previously, we would go back to `/`.